src/openms/source/ANALYSIS/QUANTITATION/IsotopeLabelingMDVs.cpp

```diff
@@ -0,0 +1,314 @@
+    const std::vector<double>& fragment_isotopomer_measured,
+    const std::vector<double>& fragment_isotopomer_theoretical)
+  {
+    feature_with_accuracy_info = normalized_feature;
+    
+    std::vector<double> fragment_isotopomer_abs_diff;
+    for (size_t i = 0; i < fragment_isotopomer_theoretical.size(); ++i) {
+      fragment_isotopomer_abs_diff.push_back(std::abs(fragment_isotopomer_theoretical[i] - fragment_isotopomer_measured[i]));
+    }
+    
+    double diff_mean = 0.0;
+    for (size_t i = 0; i < fragment_isotopomer_abs_diff.size(); ++i) {
+      diff_mean += fragment_isotopomer_abs_diff.at(i);
+    }
+    
+    diff_mean /= fragment_isotopomer_abs_diff.size();
+    
+    std::vector<double> fragment_isotopomer_abs_diff_;
+    for (size_t i = 0; i < fragment_isotopomer_abs_diff.size(); ++i) {
+      fragment_isotopomer_abs_diff_.push_back(std::abs(fragment_isotopomer_abs_diff[i] - diff_mean));
+    }
+    
+    diff_mean = 0.0;
+    for (size_t i = 0; i < fragment_isotopomer_abs_diff_.size(); ++i) {
+      diff_mean += fragment_isotopomer_abs_diff_.at(i);
+    }
+    
+    diff_mean /= fragment_isotopomer_abs_diff_.size();
+    
+    feature_with_accuracy_info = normalized_feature;
+    feature_with_accuracy_info.setMetaValue("average_accuracy", diff_mean);
+  }
+
+  void IsotopeLabelingMDVs::calculateMDVAccuracies(
+    const FeatureMap& normalized_featureMap,
+    FeatureMap& featureMap_with_accuracy_info,
+    const std::vector<double>& fragment_isotopomer_measured,
+    const std::vector<double>& fragment_isotopomer_theoretical)
+  {
+    for (const Feature& feature : normalized_featureMap) {
+      Feature feature_with_accuracy_info;
+      calculateMDVAccuracy(feature, feature_with_accuracy_info, fragment_isotopomer_measured, fragment_isotopomer_theoretical);
+      featureMap_with_accuracy_info.push_back(feature_with_accuracy_info);
+    }
+  }
+
+  void IsotopeLabelingMDVs::calculateMDV(
+    const Feature& measured_feature,
+    Feature& normalized_feature,
+    const String& mass_intensity_type,
+    const String& feature_name)
+  {
+    std::vector<Feature> measured_feature_subordinates = measured_feature.getSubordinates();
+    normalized_feature = measured_feature;
+    if (mass_intensity_type == "norm_max")
+    {
+      if (feature_name == "intensity")
+      {
+        std::vector<OpenMS::Peak2D::IntensityType> intensities_vec;
+        for (auto it = measured_feature_subordinates.begin(); it != measured_feature_subordinates.end(); it++)
+        {
+          intensities_vec.push_back(it->getIntensity());
+        }
+        std::vector<OpenMS::Peak2D::IntensityType>::iterator max_it = std::max_element(intensities_vec.begin(), intensities_vec.end());
+        double measured_feature_max = intensities_vec[std::distance(intensities_vec.begin(), max_it)];
+      
+        for (size_t i = 0; i < normalized_feature.getSubordinates().size(); ++i)
+        {
+          if(measured_feature_max != 0.0) {
```

Comment:
space after if

Reply:
why a space after if is required? in a source file you created recenlty [SimpleSearchEngineAlgorithm.cpp](https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/ANALYSIS/ID/SimpleSearchEngineAlgorithm.cpp) this convention isn't applied.